src/UnitySampleProject/Assets/Samples/web3.unity SDK Marketplace/2.6.0/Marketplace Samples/Scripts/AuthSystemManagerMarketplace.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using Newtonsoft.Json;
+using UnityEngine;
+using UnityEngine.Networking;
+using ChainSafe.Gaming.Marketplace.Models;
+
+namespace ChainSafe.Gaming.Marketplace
+{
+    /// <summary>
+    /// Auth system manager to help with refresh tokens.
+    /// </summary>
+    public static class AuthSystemManagerMarketplace
```

Comment:
Maybe exclude the manager name in here.

Reply:
Or no, why is this not a basic class that gets injected trought the DI?

Reply:
it has ambiguous naming with the response class so i made it that

Reply:
I mean you can just call this one MarketplaceAuth. And inject it as a dependency. Have the IMarketplaceAuth interface and do the standard thingy with our extendibility.